### ISSUE TITLE
Does this work with es6 syntax?

### ISSUE THREAD

**Issue by tnrich** [maxogden/dependency-check#37]
Just curious if the `import x from 'x'` syntax will work?

**Comment by maxogden** [comment]
@tnrich no idea (havent used those yet). The actual parsing happens in this module https://github.com/substack/node-detective

**Comment by rk08** [comment]
@tnrich did you test it out

**Comment by tnrich** [comment]
@rk08  it wasn't working for me, but I also had other weirdness in my code like jsx

**Comment by mattdesl** [comment]
This will work once a patch like this lands in `detective`:
https://github.com/substack/node-detective/pull/57

I haven't had time to clean up that PR.

**Comment by JamesKyburz** [comment]
How about swapping out `node-detective` with [detect-import-require]?

[detect-import-require]:http://npm.im/detect-import-require

Then both require and import would work.

**Comment by JamesKyburz** [comment]
See https://github.com/substack/node-detective/pull/57 for details.

**Comment by blakeembrey** [comment]
Closing in favour of https://github.com/maxogden/dependency-check/issues/60. When node supports ES6 modules directly, we should look at pulling in an updated `detective` to use.

**Comment by jbubsk** [comment]
Guys, so what about support es6 syntax as `import` ? There was a PR https://github.com/maxogden/dependency-check/compare/master...emkay:es-whatevs-module-support#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R16, what about merging it?
